airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/common.py

```diff
@@ -305,14 +305,14 @@ async def _build_connector_acceptance_test(self, connector_under_test_container:
         return cat_container.with_unix_socket("/var/run/docker.sock", self.context.dagger_client.host().unix_socket("/var/run/docker.sock"))
 
 
-class RegressionTests(Step):
-    """A step to run regression tests for a connector."""
+class LiveTests(Step):
```

Comment:
As `LiveTests` is a "macro" test suite composed of `ValidationTests` and `RegressionTests` I would suggest making it "abstract" and implement a specific `ValidationTests` step...

In it's current form I can't see how this step is execute a different thing than `RegressionTests` as the test input directory remains `src/live_tests/regression_tests`

Reply:
> As LiveTests is a "macro" test suite composed of ValidationTests and RegressionTests I would suggest making it "abstract" and implement a specific ValidationTests step...

Done.

> In it's current form I can't see how this step is execute a different thing than RegressionTests as the test input directory remains src/live_tests/regression_tests

Oops, that was an oversight on my part. Updated the input directory to `src/live_tests`.